LICENSE

```diff
@@ -1,21 +1,28 @@
-MIT License
+BSD 3-Clause License
```

Comment:
Is this a better license to use for Alkali?

Reply:
I'm not sure about that, but I was trying to follow more closely what it's in `package.json`